packages/application/src/domain/application/application.ts

```diff
@@ -210,7 +216,7 @@ export class DatacapAllocator extends AggregateRoot {
   }
 
   edit(file: ApplicationPullRequestFile) {
-    this.ensureValidApplicationStatus([ApplicationStatus.KYC_PHASE, ApplicationStatus.GOVERNANCE_REVIEW_PHASE])
+    //this.ensureValidApplicationStatus([ApplicationStatus.KYC_PHASE, ApplicationStatus.GOVERNANCE_REVIEW_PHASE])
```

Comment:
Are you absolutely sure about this? I think this was here to prevent things tripping over themselves when the app restarts WITHOUT wiping everything clean. Have you tested that path? If not we should do it before committing this to prod